overdrive.py

```diff
@@ -131,8 +132,6 @@ def __init__(self, collection):
         else:
             self.parent_library_id = None
 
-        self.collection_id = collection.id
```

Comment:
Here collection is the parent collection instead of the child collection. This collection id is used later to set the collection on license pools, so it should be the child collection.